### ISSUE TITLE
Failing on camelCase rule

### ISSUE THREAD

**Issue by davidyell** [cakephp/cakephp-codesniffer#108]
So I have this error.

`27 | ERROR | Member variable "Imagine" is not in valid camel caps format`

My class looks like this.

``` php
private $Imagine;
```

I'm not really sure how to make that "camel caps" whatever that might be.

PS, The reason it's not `$_Imagine` is because I've removed that rule :snake: 
Here are the changes I've made. https://github.com/UKWebMedia/cakephp-codesniffer/compare/cakephp:master...master
